Packages/manifest.json

```diff
@@ -1,9 +1,10 @@
 {
   "dependencies": {
+    "com.cdm.authentication": "https://github.com/cdmvision/authentication-unity.git#1.1.3",
```

Comment:
I have opened an issue at https://github.com/cdmvision/authentication-unity/issues/13 to add this library to OpenUPM; when that is done, we can add it as a dependency to the package itself